README.md

```diff
@@ -1,63 +1,320 @@
+## Introduction
 
-## Install
+This repo provides the code for several example dApps in the [Axelar Local Development Environment](https://github.com/axelarnetwork/axelar-local-dev). It contains both the JavaScript and the smart contract code (.sol files) for each example. To try them out:
 
-1. Run `npm update && npm install`.
-2. Run `npm run build` to compile all contracts.
+- Set up your system.
+- Deploy and test the dApps.
 
-## Examples
+Note: You may see example folders in this repo that are not described below. They are either placeholders for future dApps, such as the `temp` folder, or they are dApps in progress and we'll add a description when they're finished.
 
-To run any example on your local machine first run `node scripts/createLocal` on a separate terminal.
+## Prerequisites
 
-All examples use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` as the user that deploys and runs all tests. This address is funded by default when creating a local set of chains. Make sure it is funded on testnets as well. Run `node/printBalances` (optionally specify an address if you wish to check an address other than the one above) to check the balances on all five supported testnets of any address.
+### One-time setup
 
-## Deployment
+1. You'll need to have node.js installed to run network dApps. To make sure you have it installed, run `node -v`. If no version is returned, see
 
-Some examples need to first be deployed on the network you wish to run them on. To do this simply run `node scripts/deploy examples/${name} ${local|testnet}`. Replace `${var}` appropriately. Cannot be run for the `sent-token` and `deposit-address` examples as no deployment is needed.
+  [Nodejs.org/downloads](https://nodejs.org/en/download/).
 
-## Testing
+2. Clone the repo with:
 
-All examples can be tested with `node scripts/test/{example-name} ${local|testnet} ...params`. See below for what values params can take. All params are optional and have default values and can be ommited.
+ `git clone https://github.com/axelarnetwork/axelar-local-gmp-examples.git`.
 
-### Sent token
+### Whenever you start a new work session
 
-Run `node scripts/test examples/send-token ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+1. cd to `axelar-local-gmp-examples`.
 
-### Deposit Address
+2. Build contracts and tests (The update and install take a few minutes.):
+   ```
+   npm update && npm install
+   npm run build
+   ```
 
-Run `node scripts/test examples/deposti-address ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+3. To connect to the axelar network, run:
```

Comment:
This step should be unnecessary---please delete it.  It should already be done when you do `npm install` as part of the one-time setup.